src/js/apps/globals/nav_app.js

```diff
@@ -80,8 +80,7 @@ const patientsAppWorkflowsNav = new Backbone.Collection([
   },
   {
     text: i18n.patientsApp.worklists.schedule,
-    event: 'worklist',
-    eventArgs: ['schedule'],
+    event: 'schedule',
```

Comment:
does this work?

Reply:
nope sure didn't! Pushed up a simple fix. Lemme know if you want me to make it so the items don't require `eventArgs` at all

Reply:
I also think that `this.startRoute('schedule');` is causing an error when clicking on the schedule nav when on the page.  This is because the schedule app isn't a subrouter so it should be `this.startCurrent('schedule')`

An empty array is ok with me for now